interface/forms/button_ros/new.php

```diff
@@ -0,0 +1,667 @@
+			<input id="HEME5" name="HEME5" type="checkbox" value="on" >
+			<label for="HEME5"> <?php echo xlt('Hx Blood Transfusion');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="HEME6" name="HEME6" type="checkbox" value="on" >
+			<label for="HEME6"> <?php echo xlt('HIV+');?></label>
+		</span>
+		</TD>
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
+					<TEXTAREA NAME="HEME_notes" ROWS=3 COLS=25 WRAP=SOFT STYLE="width: 2in; height: 0.21in"></TEXTAREA>
+					</TD>
+	</TR>	
+	
+	<TR class="alt">
+	<TD>
+			<B><?php echo xlt('ENDO:');?></B>
+	</TD>
+	</TR>
+	<TR>
+	<TD>
+		<span class="checkbox-button">
+			<input id="ENDO1" name="ENDO1" type="checkbox" value="on" >
+			<label for="ENDO1"> <?php echo xlt('Polydipsia');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENDO2" name="ENDO2" type="checkbox" value="on" >
+			<label for="ENDO2"> <?php echo xlt('Heat Intolerance');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENDO3" name="ENDO3" type="checkbox" value="on" >
+			<label for="ENDO3"> <?php echo xlt('Cold Intolerance');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENDO4" name="ENDO4" type="checkbox" value="on" >
+			<label for="ENDO4"> <?php echo xlt('Excessive Sweating');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENDO5" name="ENDO5" type="checkbox" value="on" >
+			<label for="ENDO5"> <?php echo xlt('Feelings of Weakness');?></label>
+		</span>
+		</TD>
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
+					<TEXTAREA NAME="ENDO_notes" ROWS=3 COLS=25 WRAP=SOFT STYLE="width: 2in; height: 0.21in"></TEXTAREA>
+					</TD>
+	</TR>	
+	
+	<TR class="alt">
+	<TD>
+		<B><?php echo xlt('DERM:');?></B>
+	</TD>
+	</TR>
+	<TR>
+	<TD>
+		<span class="checkbox-button">
+			<input id="DERM1" name="DERM1" type="checkbox" value="on" >
+			<label for="DERM1"> <?php echo xlt('Dry Skin');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="DERM2" name="DERM2" type="checkbox" value="on" >
+			<label for="DERM2"> <?php echo xlt('Itching');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="DERM3" name="DERM3" type="checkbox" value="on" >
+			<label for="DERM3"> <?php echo xlt('Peeling Skin');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="DERM4" name="DERM4" type="checkbox" value="on" >
+			<label for="DERM4"> <?php echo xlt('Skin Scalding');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="DERM5" name="DERM5" type="checkbox" value="on" >
+			<label for="DERM5"> <?php echo xlt('Localized Skin Discoloration');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="DERM6" name="DERM6" type="checkbox" value="on" >
+			<label for="DERM6"> <?php echo xlt('Rash');?></label>
+		</span>
+		</TD>
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
```

Comment:
That was because the reviewer was not a technical writer.
You never, ever, ever do a translation or an auto-linked definition of an initialization or a acronym without having a special flag in the function, or cast it with punctuation or other markups to indicate that.  Imagine translating GRITS.    Maiz Molido isn't the same.
.Ever.